frontend/app/src/next/components/runs/runs-badge.tsx

```diff
@@ -0,0 +1,137 @@
+import { Badge, BadgeProps } from '@/next/components/ui/badge';
+import { V1TaskStatus, WorkflowRunStatus } from '@/lib/api';
+import { cn } from '@/next/lib/utils';
+
+interface RunsBadgeProps extends BadgeProps {
+  status?: StatusKey;
+  count?: number;
+  percentage?: number;
+  animated?: boolean;
+  isLoading?: boolean;
+}
+
+type StatusConfig = {
+  colors: string;
+  primary: string;
+  primaryOKLCH: string;
+  label: string;
+};
+
+type StatusKey = WorkflowRunStatus | V1TaskStatus | 'QueueMetrics';
+
+export const RunStatusConfigs: Record<StatusKey, StatusConfig> = {
+  [WorkflowRunStatus.RUNNING]: {
+    colors:
+      'text-indigo-800 dark:text-indigo-300 bg-indigo-500/20 ring-indigo-500/30',
+    primary: 'text-indigo-500 bg-indigo-500',
+    primaryOKLCH: 'oklch(0.585 0.233 277.117)',
```

Comment:
@mrkaye97 @abelanger5 what in the world is an OKLCH

Reply:
heh I think it's definitely gpt that did this, but it's not hallucination - it's a different way of specifying colors that's more granular or something idk